src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -5570,6 +5570,65 @@ public void archiveVersion(Long id) {
             }
         }
     }
+    
+    public boolean isArchivable() {
+        String className = settingsWrapper.getValueForKey(SettingsServiceBean.Key.ArchiverClassName, null);
```

Comment:
Does it make sense for this method (and the next one) to be on the ArchiverUtil?

Reply:
@sekmiller  - Maybe? These, and the one above are all used in the dataset-versions.xhtml file so moving them completely to ArchiverUtil (or DatasetServiceBean above) means having to make those classes (with ArchiverUtil not being a bean) accessible in the xhtml and not being able to use settingsWrapper to cache the property values(again because , with ArchiverUtil is not a bean/ is called from the API where there is no view/session scope). 

One way I could simplify a bit would be to shift the reflective call logic into ArchiverUtil, e.g. a getArchiverOption(String className, String methodToInvoke) method. That would reduce duplicate code and minimize what's in DatasetPage. 

Does that make sense? Or is it worth moving them out of DatasetPage completely?

Reply:
Maybe just isSomeVersionArchived which is in the DatasetPage as well as Admin and Datasets.

Reply:
or maybe it's not worth the trouble

Reply:
I'll do some quick de-duplication and push it back. Thanks!

Reply:
I moved the guts of someVersionArchived to the util but the rest seem hard - there are methods that require a dataset and some that are static, some use settingsWrapper and some settingsServiceBean due to their context (UI or API) so as I tried to refactor I didn't see good opportunities to consolidate.